code/stable/gamephys/SRS/Chipmunk_SRS.tex

```diff
@@ -304,7 +304,7 @@ \subsubsection{Terminology and Definitions}
 \end{itemize}
 \subsubsection{Goal Statements}
 \label{Sec:GoalStmt}
-Given the inputs, the goal statements are:
+Given the kinematic properties, and forces including any (collision forces) applied on a set of rigid bodies, the goal statements are:
```

Comment:
Let us change `kinematic properties` to `current kinematic state`.  We should also add mass and gravitational field to the list of inputs.  These will all be nicely refined by the instance models.